docs-new/docs/graphql/core/api-reference/syntax-defs.mdx

```diff
@@ -766,232 +700,181 @@ String: String } } \]
-| function         | true     | `FunctionName <FunctionName>` | The SQL function                                                                                                                                        |
+| function         | true     | [FunctionName](#functionname) | The SQL function                                                                                                                                        |
 | table_argument   | false    | String                        | Name of the argument which accepts a table row type. If omitted, the first argument is considered a table argument                                      |
 | session_argument | false    | String                        | Name of the argument which accepts the Hasura session object as a JSON/JSONB value. If omitted, the Hasura session object is not passed to the function |
 
-## Function Configuration
+## Function Configuration {#functionconfiguration}
 
 | Key                | Required | Schema                                                      | Description                                                                                                                                             |
 |--------------------|----------|-------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
 | custom_name        | false    | `String`                                                    | Customise the `<function-name>` with the provided custom name value. The GraphQL nodes for the function will be generated according to the custom name. |
-| custom_root_fields | false    | `Custom Function Root Fields <custom_function_root_fields>` | Customise the root fields                                                                                                                               |
+| custom_root_fields | false    | [Custom Function Root Fields](#custom-function-root-fields) | Customise the root fields                                                                                                                               |
 | session_argument   | false    | <span class="title-ref">String</span>                       | Function argument which accepts session info JSON                                                                                                       |
 | exposed_as         | false    | <span class="title-ref">String</span>                       | In which part of the schema should we expose this function? Either "mutation" or "query".                                                               |
 
-<div id="function_req_note">
-
-<div class="note">
-
-<div class="title">
-
-Note
-
-</div>
+:::info Note
```

Comment:
missed id for the note

an empty div before this line with an id should do

`<div id="function-req-note" />`

Reply:
Fixed.